flexget/utils/imdb.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
         self.min_diff = 0.01
         self.debug = False
 
-        self.max_results = 10
+        self.max_results = 50
```

Comment:
The test i added is just number 13, not sure if any other movies in my collection get ranked lower in the search results page, lower it to 20?

About making it configurable... Well.. same problem as the geolocation nuke... I don't know how to add a config option for utils/imdb.py, it's not a plugin. If you give me instructions on how it should be done, sure, i will add it and also make the geolocation ip configurable while at it.